Keda.Cosmosdb.Scaler/src/Services/CosmosDBConnectionString.cs

```
@@ -0,0 +1,32 @@
+﻿using Keda.Cosmosdb.Scaler;
+using System;
+using System.Data.Common;
+
+namespace Keda.CosmosDB.Scaler.Services
+{
+    internal class CosmosDBConnectionString
+    {
+        public CosmosDBConnectionString(string connectionString)
+        {
+            DbConnectionStringBuilder builder = new DbConnectionStringBuilder
+            {
+                ConnectionString = connectionString
+            };
+
+            object key = null;
+            if (builder.TryGetValue(Constants.AccountKey, out key))

```

Maybe it should throw an exception if it's not there?